packages/contracts-bedrock/scripts/Types.sol

```diff
@@ -13,6 +13,7 @@ library Types {
         address OptimismMintableERC20Factory;
         address OptimismPortal;
         address OptimismPortal2;
+        address OptimismPortalInterop;
```

Comment:
We should be able to get rid of this diff, either interop or the portal is deployed. For all intents and purposes, the interop portal is the portal